ローカルでpnpm run lintするとエラーが出るのにGithub Workflowではエラーが出てない

## 不具合の内容

タイトルのとおりです。

原因がわからずにいます。
[workflowのログ](https://github.com/VOICEVOX/voicevox/actions/runs/13976852430/job/39132656646)を見ても別にスキップされてるわけではなさそう。

### 現象・ログ

ローカルで実行するとこうなります。
`rm .eslintcache`してから実行しても同じ結果でした。
<details>

```bash

> voicevox@999.999.999 lint /Users/[name]/Github/voicevox
> cross-env ESLINT_FILE_PROGRESS=1 eslint --cache


/Users/[name]/Github/voicevox/tests/unit/backend/browser/fakePath.spec.ts
  3:1   error  Unsafe call of a(n) `error` type typed value                            @typescript-eslint/no-unsafe-call
  3:1   error  Unsafe call of a(n) `error` type typed value                            @typescript-eslint/no-unsafe-call
  8:35  error  Unsafe argument of type `any` assigned to a parameter of type `string`  @typescript-eslint/no-unsafe-argument

/Users/[name]/Github/voicevox/tests/unit/backend/common/configManager.spec.ts
   45:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   48:1   error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
   49:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   49:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   49:51  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
   52:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   52:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   52:49  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
   61:1   error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
   62:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   62:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   62:51  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
   65:9   error  Unsafe assignment of an error typed value                                                                                                                           @typescript-eslint/no-unsafe-assignment
   65:19  error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   65:19  error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   67:6   error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
   73:9   error  Unsafe assignment of an error typed value                                                                                                                           @typescript-eslint/no-unsafe-assignment
   73:29  error  Unsafe member access .mock on an `error` typed value                                                                                                                @typescript-eslint/no-unsafe-member-access
   74:20  error  Unsafe member access .__internal__ on an `error` typed value                                                                                                        @typescript-eslint/no-unsafe-member-access
   78:3   error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
   79:5   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   79:5   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   79:53  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
   82:5   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   82:5   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   82:51  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
   85:5   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   85:5   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
   85:51  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
   98:1   error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  100:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  100:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  100:51  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
  103:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  103:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  103:49  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
  106:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  106:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  106:49  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
  157:1   error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  158:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  158:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  158:51  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
  161:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  161:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  161:49  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
  164:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  164:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  164:49  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
  176:1   error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  177:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  177:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  177:51  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
  180:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  180:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  180:49  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access
  183:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  183:3   error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  183:49  error  Unsafe member access .mockImplementation on an `error` typed value                                                                                                  @typescript-eslint/no-unsafe-member-access

/Users/[name]/Github/voicevox/tests/unit/composable/useFetchNewUpdateInfos.spec.ts
  15:3  error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  22:9  error  Unsafe call of a(n) `error` type typed value                                                                                                                        @typescript-eslint/no-unsafe-call
  27:1  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  44:1  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises

/Users/[name]/Github/voicevox/tests/unit/composable/useModifierKey.spec.ts
  21:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  39:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises

/Users/[name]/Github/voicevox/tests/unit/domain/defaultEngine/latetDefaultEngine.spec.ts
   7:9   error  Unsafe assignment of an error typed value                          @typescript-eslint/no-unsafe-assignment
   7:15  error  Unsafe call of a(n) `error` type typed value                       @typescript-eslint/no-unsafe-call
   7:15  error  Unsafe call of a(n) `error` type typed value                       @typescript-eslint/no-unsafe-call
   9:6   error  Unsafe member access .mockResolvedValue on an `error` typed value  @typescript-eslint/no-unsafe-member-access
  15:3   error  Unsafe call of a(n) `error` type typed value                       @typescript-eslint/no-unsafe-call
  15:7   error  Unsafe member access .mockRestore on an `error` typed value        @typescript-eslint/no-unsafe-member-access

/Users/[name]/Github/voicevox/tests/unit/domain/japanese.spec.ts
   8:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  14:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises

/Users/[name]/Github/voicevox/tests/unit/domain/sing/applyPhonemeTimingEdit.spec.ts
   52:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
   63:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
   83:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  104:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises

/Users/[name]/Github/voicevox/tests/unit/domain/sing/shouldPlayTracks.spec.ts
  20:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  35:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises

/Users/[name]/Github/voicevox/tests/unit/store/utility.spec.ts
  110:5  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  119:5  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  132:5  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  145:5  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  161:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  194:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  225:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
  238:3  error  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises

✖ 90 problems (90 errors, 0 warnings)

 ELIFECYCLE  Command failed with exit code 1.
```

</details>

> ✖ 90 problems (90 errors, 0 warnings)

### 再現手順

ローカルで`pnpm run lint`

### 期待動作

workflowでも同じエラーが出る

## VOICEVOXのバージョン

4790480bba20e05f8bab52d369b52437ac4427dc

## その他

<!-- 関連して何か気がついたこと、気になることがあればココに書いてください -->

--- a/tests/unit/backend/browser/fakePath.spec.ts
+++ b/tests/unit/backend/browser/fakePath.spec.ts
@@ -1,3 +1,4 @@
+import { it, expect } from "vitest";
 import { isFakePath, createFakePath } from "@/backend/browser/fakePath";
 
 it.each([

--- a/tests/unit/backend/common/configManager.spec.ts
+++ b/tests/unit/backend/common/configManager.spec.ts
@@ -1,3 +1,4 @@
+import { vi, it, expect, afterEach } from "vitest";
 import pastConfigs from "./pastConfigs";
 import configBugDefaultPreset1996 from "./pastConfigs/0.19.1-bug_default_preset.json";
 import { BaseConfigManager } from "@/backend/common/ConfigManager";

--- a/tests/unit/backend/electron/VvppManager.node.spec.ts
+++ b/tests/unit/backend/electron/VvppManager.node.spec.ts
@@ -1,7 +1,7 @@
 import fs from "fs";
 import os from "os";
 import path from "path";
-import { beforeEach, expect, test } from "vitest";
+import { beforeAll, afterAll, beforeEach, expect, test } from "vitest";
 import { createVvppFile } from "./helper";
 import { EngineId, MinimumEngineManifestType } from "@/type/preload";
 import VvppManager from "@/backend/electron/manager/vvppManager";

--- a/tests/unit/backend/electron/fileHelper.node.spec.ts
+++ b/tests/unit/backend/electron/fileHelper.node.spec.ts
@@ -1,7 +1,7 @@
 import fs from "fs";
 import path from "path";
 import os from "os";
-import { test, expect, beforeAll, afterAll } from "vitest";
+import { beforeAll, afterAll, describe, expect, test } from "vitest";
 import { writeFileSafely } from "@/backend/electron/fileHelper";
 import { uuid4 } from "@/helpers/random";
 

--- a/tests/unit/backend/electron/vvppFile.node.spec.ts
+++ b/tests/unit/backend/electron/vvppFile.node.spec.ts
@@ -1,7 +1,7 @@
 import os from "os";
 import path from "path";
 import fs from "fs";
-import { test, afterAll, beforeAll } from "vitest";
+import { afterAll, beforeAll, expect, test } from "vitest";
 import { createVvppFile } from "./helper";
 import { VvppFileExtractor } from "@/backend/electron/vvppFile";
 import { uuid4 } from "@/helpers/random";

--- a/tests/unit/components/help/HelpMarkdownViewSection.spec.ts
+++ b/tests/unit/components/help/HelpMarkdownViewSection.spec.ts
@@ -1,5 +1,5 @@
 import { mount, flushPromises } from "@vue/test-utils";
-import { describe, it } from "vitest";
+import { describe, it, expect } from "vitest";
 import HelpMarkdownViewSection from "@/components/Dialog/HelpDialog/HelpMarkdownViewSection.vue";
 import { markdownItPlugin } from "@/plugins/markdownItPlugin";
 

--- a/tests/unit/composable/useFetchNewUpdateInfos.spec.ts
+++ b/tests/unit/composable/useFetchNewUpdateInfos.spec.ts
@@ -1,3 +1,4 @@
+import { vi, it, expect } from "vitest";
 import { Ref } from "vue";
 import { UpdateInfo, UrlString } from "@/type/preload";
 import { useFetchNewUpdateInfos } from "@/composables/useFetchNewUpdateInfos";

--- a/tests/unit/composable/useModifierKey.spec.ts
+++ b/tests/unit/composable/useModifierKey.spec.ts
@@ -1,3 +1,4 @@
+import { it, expect, describe } from "vitest";
 import { mount } from "@vue/test-utils";
 import { Ref } from "vue";
 import {

--- a/tests/unit/domain/defaultEngine/latetDefaultEngine.spec.ts
+++ b/tests/unit/domain/defaultEngine/latetDefaultEngine.spec.ts
@@ -1,3 +1,4 @@
+import { vi, expect, test } from "vitest";
 import latestDefaultEngineInfos from "./latestDefaultEngineInfos.json";
 import { fetchLatestDefaultEngineInfo } from "@/domain/defaultEngine/latetDefaultEngine";
 

--- a/tests/unit/domain/hotkeyAction.spec.ts
+++ b/tests/unit/domain/hotkeyAction.spec.ts
@@ -1,3 +1,4 @@
+import { expect, test } from "vitest";
 import {
   getDefaultHotkeySettings,
   hotkeyActionNameSchema,

--- a/tests/unit/domain/japanese.spec.ts
+++ b/tests/unit/domain/japanese.spec.ts
@@ -1,3 +1,4 @@
+import { it, expect, describe, test } from "vitest";
 import {
   createKanaRegex,
   convertHiraToKana,

--- a/tests/unit/domain/project.spec.ts
+++ b/tests/unit/domain/project.spec.ts
@@ -1,4 +1,5 @@
 import fs from "fs";
+import { beforeEach, describe, expect, test } from "vitest";
 import { migrateProjectFileObject } from "@/domain/project";
 import { EngineId, SpeakerId, StyleId } from "@/type/preload";
 import { resetMockMode } from "@/helpers/random";

--- a/tests/unit/domain/sing/applyPhonemeTimingEdit.spec.ts
+++ b/tests/unit/domain/sing/applyPhonemeTimingEdit.spec.ts
@@ -1,3 +1,4 @@
+import { it, expect, describe } from "vitest";
 import { uuid4 } from "@/helpers/random";
 import { FramePhoneme } from "@/openapi";
 import {

--- a/tests/unit/domain/sing/shouldPlayTracks.spec.ts
+++ b/tests/unit/domain/sing/shouldPlayTracks.spec.ts
@@ -1,3 +1,4 @@
+import { it, expect, describe } from "vitest";
 import { uuid4 } from "@/helpers/random";
 import { createDefaultTrack, shouldPlayTracks } from "@/sing/domain";
 import { Track } from "@/store/type";

--- a/tests/unit/lib/cloneWithUnwrapProxy.spec.ts
+++ b/tests/unit/lib/cloneWithUnwrapProxy.spec.ts
@@ -1,4 +1,4 @@
-import { test } from "vitest";
+import { expect, test } from "vitest";
 import { cloneWithUnwrapProxy } from "@/helpers/cloneWithUnwrapProxy";
 
 const original = { a: 1, b: { c: 2 } };

--- a/tests/unit/mock/engineMock/index.spec.ts
+++ b/tests/unit/mock/engineMock/index.spec.ts
@@ -1,3 +1,4 @@
+import { beforeEach, describe, expect, test } from "vitest";
 import { hash } from "../../utils";
 import { resetMockMode } from "@/helpers/random";
 import { createOpenAPIEngineMock } from "@/mock/engineMock";

--- a/tests/unit/store/command.spec.ts
+++ b/tests/unit/store/command.spec.ts
@@ -1,3 +1,4 @@
+import { beforeEach, expect, test } from "vitest";
 import { store } from "@/store";
 import { AudioKey } from "@/type/preload";
 import { resetMockMode, uuid4 } from "@/helpers/random";

--- a/tests/unit/store/immerPatchUtility.spec.ts
+++ b/tests/unit/store/immerPatchUtility.spec.ts
@@ -1,3 +1,4 @@
+import { describe, expect, test } from "vitest";
 import { enableMapSet, enablePatches, Immer, Patch } from "immer";
 import { applyPatches } from "@/store/immerPatchUtility";
 

--- a/tests/unit/store/singing.spec.ts
+++ b/tests/unit/store/singing.spec.ts
@@ -1,3 +1,4 @@
+import { beforeEach, expect, test } from "vitest";
 import { store } from "@/store";
 import { TrackId } from "@/type/preload";
 import { resetMockMode, uuid4 } from "@/helpers/random";

--- a/tests/unit/store/utility.spec.ts
+++ b/tests/unit/store/utility.spec.ts
@@ -1,3 +1,4 @@
+import { it, expect, describe, test } from "vitest";
 import { AccentPhrase, Mora } from "@/openapi";
 import {
   CharacterInfo,

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -16,7 +16,6 @@
     "resolveJsonModule": true,
     "sourceMap": true,
     "baseUrl": ".",
-    "types": ["vitest/globals"],
     "paths": {
       "@/*": ["src/*"]
     },

